### DISCUSSION TITLE
How to get in touch?

### DISCUSSION THREAD

**Discussion by daproject85** [EA31337/EA31337#254] [Q&A] [answered]
HI, is there a way to get in touch with you regarding this EA, MQL5 and some general github development questions with respect to MT5?

**Comment by kenorb** [comment, marked as answer]
You can join the Telegram discussion group at https://t.me/EA31337

**Reply by daproject85** [reply]
I did join it, however its like a news feed, I am unable to type anything

**Reply by kenorb** [reply]
Try clicking on the _Discuss_ button at the bottom.
